### ISSUE TITLE
How to clear down nvAlt to start afresh

### ISSUE THREAD

**Issue by johnnypoll** [ttscoff/nv#493]
If I create a new user on my Mac (High Sierra) I can create start nvAlt fine with the default initial notes appearing.

If I use my existing user, despite deleting all the following:

~/Library/Application Support/nvALT
~/Library/Application Support/Notational Data
~/Library/Preferences/net.elasticthreads.nv.plist

I can't start nvAlt, I get the dreaded message about Interim Note-Changes file could not be initialized. Then if I click Choose another folder, no window appears. If I go to nvAlt Preferences and click on Other.. folder again nothing appears.

Thanks for any insights, just want to be able to clear down and start again.

**Comment by johnnypoll** [comment]
I solved this by running the /Applications/nvALT.app/Contents/MacOS/nvAlt directly in Terminal and observing the error messages telling my my Caches folder was not writeable. Would be great if apps could display the actual underlying error in the interface, though this is a general dig at all apps worldwide, not nvalt specifically.

**Comment by kraigh** [comment]
@johnnypoll can you share exactly how you fixed this? I just updated to High Sierra and my nvAlt won't open and I feel lost and alone without it!

**Comment by johnnypoll** [comment]
Hi Kraig,

Here are my notes (extracted from my own nvAlt notes of course!) from that worrying time and how I resolved it….

Jan 2018, having a horrible time getting nvALT to run after transferring to new macbook pro.

I couldn’t even clear down the database and Preferences files and start afresh, getting an error about nvAlt not being able to create an Interim notes file. I finally worked out what was wrong by running nvAlt in Terminal and looking at log output:

/Applications/nvALT.app/Contents/MacOS/nvAlt

which informed me ~/Library/Caches was not writeable and sure enough it was owned by root, not sure why, so I fixed with:

sudo chown john Caches

then was able to run and set preferences for Text based files and to sync with SimpleNote, which nicely picked up all my notes. yay!

John
